fabric_client/README.md

```diff
@@ -0,0 +1,48 @@
+# Switchtec Fabric DMA Driver Client Examples
+
+## Summary
+
+There are two Switchtec fabric DMA clients in this directory for people to
+understand the proprietary fabric APIs of Switchtec DMA driver. They work in
+the initiator and target mode, which is considered the most common use case of
+Switchtec fabric DMA. The initiator produces data and copy them from local host
+to the buffers of a peer host. The target on the peer host consumes the data in
+one way or another.
+
+Note: this is only for reference. A real world client would be much complicated
```

Comment:
A real world client would be much more complicated with error handling and protection, and could act as initiator and target at the same time.